src/App/Lobby/LobbyService.ts

```diff
@@ -96,9 +96,7 @@ export class LobbyService {
   public static getCredential(roomID: string): IPlayerCredential | undefined {
     // return an empty IPlayerInRoom object if the player's identity is for another room
     const credentials: IStoredCredentials = JSON.parse(localStorage.getItem(FBG_CREDENTIALS_KEY));
-    if (credentials) {
-      return credentials[roomID];
-    }
+    return credentials[roomID];
```

Comment:
Isn't this going to break if localStorage is empty?

Reply:
I thought so too, but nonexistent keys are undefined.

```javascript
var x = {foo: 'bar'}
    undefined

x['foo']
    "bar"

x['bar']
    undefined

```

Reply:
But you are assuming that JSON.parse will return an object. It might return null/undefined instead, and then you get a error trying to access a key of null?

Reply:
I didn't think of that.  You are right.  👍